operations/finance/spending-company-money/staff-member-expenses/internship-expense-policy.md

```diff
@@ -0,0 +1,27 @@
+# Internship Expense Policy
+
+At Mattermost, we want our interns to be successful from Day 1 and you'll be provisioned an Expensify account. As a general rule, our [expense policy](https://handbook.mattermost.com/operations/finance/spending-company-money/expense-policy) for our full-time and part-time staff members is the same for our interns with some modifications. As an intern, you can refer to our How to spend company money policy for guidance for most expense-related questions.
```

Comment:
The "expense policy" URL isn't working. Is it still to be added? Or will it be this one: https://github.com/mattermost/mattermost-handbook/pull/501/?

The "How to spend company money" doesn't have a link. Will a link to the new page be added or should we reference the current one?

Reply:
@justinegeffen that is correct that #501 is the PR for the expense policy page. The How to Spend Company Money will be migrated to the Expense Policy Page.